charts/openshift-etcd-backup/templates/cronjob.yaml

```
@@ -0,0 +1,69 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "openshift-etcd-backup.fullname" . }}
+spec:
+  schedule: {{ .Values.backup.schedule | quote }}
+  jobTemplate:
+    spec:
```

Since we have feedback from Cyrill that the `backoffLimit` has been added for a specific reason we should add it again, but this time with a comment explaining why.
```suggestion
    spec:
      # Retrying this job is not considered safe, because of that we fail the
      # job immediately if the first execution fails.
      backoffLimit: 0
```

---

https://github.com/adfinis-sygroup/helm-charts/pull/287/commits/87c794a9293c1a97f684ea5f3506090d17582a1b